### PR TITLE
Use the correct palette for some death sequences

### PR DESCRIPTION
**Pull request by abcdefg30**
Depends on #4.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Rebased.

**Comment by GraionDilach** [comment]
DeathType3 is again ExplosionDeath and S_BANG34. :D

**Comment by abcdefg30** [comment]
Thanks, I'll update this tomorrow.

**Comment by GraionDilach** [comment]
You could also enable nukedie.shp as RadiationDeath as part of this, since we have the Desolator already.

Although that was death type 7 in RA2 with Yuri's deployweapon being 6...

**Comment by GraionDilach** [comment]
Again, see http://modenc.renegadeprojects.com/InfDeath and https://github.com/Phrohdoh/oramod-ra2/blob/master/extra-files/rules.ini#L591

**Comment by abcdefg30** [comment]
> You could also enable nukedie.shp as RadiationDeath as part of this, since we have the Desolator already.

This is already done with the 4th death type.

**Comment by GraionDilach** [comment]
Oh, sorry then, noidea why I skipped that - then only ExplosiveDeath and HeadPop is missing - and the latter ain't even needed at this point.

**Comment by abcdefg30** [comment]
I pushed an update but didn't get to fix die3 yet.

**Comment by phrohdoh** [comment]
Let's not sneak/request more things in PRs than what the title/OP doesn't cover.

**Comment by abcdefg30** [comment]
Ok, I'll add the other death types in another PR then.

**Comment by phrohdoh** [comment]
Not sure about this, when I kill infantry now with `shk` they turn into clouds of gray/green smoke.
Is this vanilla RA2 behaviour? Please confirm @GraionDilach, if possible.

#4 needs fixing, so this can't be merged until then. Thanks for your efforts and ping me when I can look at this again. :smile:

**Comment by GraionDilach** [comment]
You're the first person whom I heard complaining that the effect uses dark green as the blackish color - on the other hand I acknowledge, the original game looped the first four frames three times before playing the fall-apart-into-dust subsequence, which probably makes this more noticable.

So I'd reword the question - would you want to see an amendment which fixes the colors or not?

**Comment by GraionDilach** [comment]
If the answer is yes to the above, then I'll gladly file a follow-up resolving that, ofc.

**Comment by phrohdoh** [comment]
What would the fixed colors be?
I don't see logic in an organic substance being zapped and turning into a smoke cloud similar to the grizzly's projectile's puff.

I'd like the colors and anims to match vanilla RA2, even if I find them displeasing.

**Comment by phrohdoh** [comment]
This needs to be rebased.

**Comment by abcdefg30** [comment]
Updated.

**Comment by GraionDilach** [comment]
I'd prefer having a dark blue tint instead of the green, but your call.

Also, FYI, RA1 tesla death was pretty much the same with more smoke. :P

**Comment by phrohdoh** [comment]
Well now the cloud doesn't appear (which is only logical to me), so I'm fine with this, thanks!
